be/src/pipeline/task_scheduler.h

```diff
@@ -66,8 +66,8 @@ class TaskScheduler {
 private:
     std::unique_ptr<ThreadPool> _fix_thread_pool;
     std::shared_ptr<TaskQueue> _task_queue;
-    std::vector<std::unique_ptr<std::atomic<bool>>> _markers;
-    std::atomic<bool> _shutdown;
+    std::vector<bool> _markers;
+    bool _shutdown;
```

Comment:
need init this variable

Reply:
![image](https://github.com/apache/doris/assets/10553413/e488c664-330b-4f4a-b247-7465b50880ac)
construct func do the work